packages/components/molecules/f-restaurant-card/README.md

```diff
@@ -0,0 +1,127 @@
+<div align="center">
+
+# f-restaurant-card
+
+<img width="125" alt="Fozzie Bear" src="../../../../bear.png" />
+
+ Responsible for displaying restaurant data and linking to a restaurant
+
+</div>
+
+---
+
+[![npm version](https://badge.fury.io/js/%40justeat%2Ff-restaurant-card.svg)](https://badge.fury.io/js/%40justeat%2Ff-restaurant-card)
+[![CircleCI](https://circleci.com/gh/justeat/fozzie-components.svg?style=svg)](https://circleci.com/gh/justeat/workflows/fozzie-components)
+[![Coverage Status](https://coveralls.io/repos/github/justeat/f-restaurant-card/badge.svg)](https://coveralls.io/github/justeat/f-restaurant-card)
+[![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-restaurant-card/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-restaurant-card?targetFile=package.json)
+
+---
+
+## Usage
+
+### Installation
+
+Install the module using npm or Yarn:
+
+```sh
+yarn add @justeat/f-restaurant-card
+```
+
+```sh
+npm install @justeat/f-restaurant-card
+```
+
+
+
+### Vue Applications
+
+You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+
+```js
+import RestaurantCard from '@justeat/f-restaurant-card';
+import '@justeat/f-restaurant-card/dist/f-restaurant-card.css';
+
+export default {
+    components: {
+        RestaurantCard
+    }
+}
+```
+
+If you are using Webpack, you can import the component dynamically to separate the `restaurant-card` bundle from the main `bundle.client.js`:
+
+```js
+import '@justeat/f-restaurant-card/dist/f-restaurant-card.css';
```

Comment:
Shall we add a comment to suggest for components used in more than one location to have the styles imported in a _once, single place in the host app_. e.g. Host app `main.scss`

Reply:
I'd suggest making that change in the Yeoman generator! Sounds like a good idea. I plan to update this readme once we start properly adding code to the component and will include this suggestion :)